INSTALL.rst

```diff
@@ -84,17 +84,42 @@ Installing from source
 
 If you are interested in contributing to Matplotlib development,
 running the latest source code, or just like to build everything
-yourself, it is not difficult to build Matplotlib from source.  Grab
-the latest *tar.gz* release file from `the PyPI files page
-<https://pypi.org/project/matplotlib/>`_, or if you want to
-develop Matplotlib or just need the latest bugfixed version, grab the
-latest git version, and see :ref:`install-from-git`.
+yourself, it is not difficult to build Matplotlib from source.
 
-Matplotlib can be installed from the source directory with a simple ::
+The easiest way to get the latest development version to start contributing
+is to go to the git `repository <https://github.com/matplotlib/matplotlib>`_
+and run: ::
 
-  python -m pip install .
+  git clone https://github.com/matplotlib/matplotlib.git
 
-We provide a setup.cfg_ file which you can use to customize the build
+If you're developing, it's better to do it in editable mode. The reason why
+is that pytest's test discovery only works for Matplotlib
+if installation is done this way. Also, editable mode allows your code changes
+to be instantly propagated to your library code without reinstalling (though
+you will have to restart your python process / kernel): ::
+
+  pip install -e .
```

Comment:
We've changed from `pip install` to `python -m pip install` throughout (https://github.com/matplotlib/matplotlib/pull/9291#discussion_r143268629). Is that not needed anymore?

Reply:
They are unless you have an environment where you didn’t install pip.  In that case if you just call pip it may call a pip that is not compatible with the python of your current environment and spurious issue reports ensue.

Reply:
We should stick with `python -m pip`, I'm going to push a commit making that change.